lazy_tensor_core/OP_LOWERING_GUIDE.md

```diff
@@ -14,11 +14,11 @@ You can find the definition of the C++ ATen operations in [native_functions.yaml
 ## File structure
 Files mentioned below live under the `lazy_tensor_core/lazy_tensor_core/csrc` and `lazy_tensor_core/lazy_tensor_core/csrc/ts_backend` folders, with the exception of `ts_native_functions.yaml`
 
-1. `ts_native_functions.yaml` contains the list of all operators that are lowered. Each operator name must directly match a pytorch operator listed in [native_functions.yaml](https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/native_functions.yaml). This file serves as the interface to adding support for new operators, and is an input to PyTorch's [codegen machinery](https://github.com/pytorch/pytorch/blob/master/tools/codegen/gen_backend_stubs.py). It generates the below 3 files: `XLANativeFunctions.h`, `RegisterXLA.cpp`, and `RegisterAutogradXLA.cpp`
-1. `XLANativeFunctions.h` and `aten_xla_type.cpp` are entry points of PyTorch to the lazy tensors world, and contain the manually written lowerings for each operator. `XLANativeFunctions.h` is auto-generated through a combination of `ts_native_functions.yaml` and the PyTorch core `native_functions.yaml` file, and contains declarations for kernels that need to be defined in `aten_xla_type.cpp`. The kernels written here need to construct `LazyTensor`s using the input `at::Tensor` and other parameters. The resulting `LazyTensor` needs to be converted back to the `at::Tensor` before returning to the PyTorch world.
-1. `RegisterXLA.cpp` and `RegisterAutogradXLA.cpp` are auto-generated files that register all lowerings to the PyTorch Dispatcher. They also include auto-generated wrapper implementations of `out=` and `inplace` operators.
+1. `ts_native_functions.yaml` contains the list of all operators that are lowered. Each operator name must directly match a pytorch operator listed in [native_functions.yaml](https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/native_functions.yaml). This file serves as the interface to adding support for new operators, and is an input to PyTorch's [codegen machinery](https://github.com/pytorch/pytorch/blob/master/tools/codegen/gen_backend_stubs.py). It generates the below 3 files: `LazyNativeFunctions.h`, `RegisterXLA.cpp`, and `RegisterAutogradLazy.cpp`
```

Comment:
Should we rename `RegisterXLA.cpp` and `aten_xla_type.cpp` as well?

Reply:
Done.